pallets/circuit/src/lib.rs

```diff
@@ -1249,10 +1249,6 @@ impl<T: Config> Pallet<T> {
                     .flat_map(|(cnt, fsx_step)| {
                         fsx_step
                             .iter()
-                            .map(|full_side_effect| full_side_effect.clone())
```

Comment:
@MaciejBaj I removed the local filter here, because I use the mapping for lookups in confirm and bid